src/main/java/org/zalando/nakadi/repository/db/EventTypeDbRepository.java

```diff
@@ -31,14 +35,18 @@ public EventTypeDbRepository(final JdbcTemplate jdbcTemplate, final ObjectMapper
     }
 
     @Override
+    @Transactional
```

Comment:
Added to remove fd4709b

My goal with this annotation is to make sure that we do not have "half-done" operations. I mean, that insertion into event_type works but then insertion into "schema" fails. With this annotation it should commit only after both operations are done or none (rollback).